draft-ietf-quic-transport.md

```diff
@@ -2403,6 +2403,15 @@ that it sends.
 Strategies and implications of the frequency of generating acknowledgments are
 discussed in more detail in {{QUIC-RECOVERY}}.
 
+# Special Considerations for Packetization Layer PMTU Discovery
+
+The PADDING frame provides a useful option for PMTU probe packets that does not
+exist in other transports. QUIC Packets with no frames other than PADDING must be
+acknowledged, but need not be retransmitted if lost. PADDING probe packets, if
+lost, also do not delay delivery of application data. Their only impact on
```

Comment:
I would remove the "do not delay clause" on the basis that they could consume congestion control bytes.

Reply:
Perhaps poorly worded, but the subject of the setting is "the <b>loss</b> of PADDING probe packets". Sending them in the first place consumes cwnd, I'll reorder the sentences.